xercesc.sh

```diff
@@ -0,0 +1,33 @@
+package: XercesC
+version: v3.1.4
+source: https://github.com/PMunkes/XercesC
+tag: v3.1.4
+env:
+  XERCESC_INST_DIR: "$XERCESCINST"
```

Comment:
I am currently testing a build without this variable, as I think this is no longer needed. It was needed in a previous version of the patch, but has since been superseded by the use of `$XERCESC_ROOT` at all relevant positions.